python/ray/data/dataset.py

```diff
@@ -2513,7 +2513,7 @@ def write_json(
         ray_remote_args: Dict[str, Any] = None,
         **pandas_json_args,
     ) -> None:
-        """Writes the :class:`~ray.data.Dataset` to JSON files.
+        """Writes the :class:`~ray.data.Dataset` to JSON and JSONL files.
```

Comment:
Looks like we use Pandas' `to_json`, and that always writes to JSON (not JSONL)?

```suggestion
        """Writes the :class:`~ray.data.Dataset` to JSON files.
```

Reply:
I tried `ds.write_json("xxx.jsonl")` locally, and it did write jsonl files. But I'm not sure it's because the ".jsonl" suffix or something else that makes it to write jsonl files. I think we need to figure it out and also make it clear in the doc.

Reply:
It's always writing in JSONL format. If it only has a single line, it becomes a JSON file. More context in https://github.com/ray-project/ray/pull/37637#issuecomment-1646104090 .

Reply:
Got it. let's update the doc to reflect this?

Reply:
Sure, updated.